docs/tools/Renderware/common/files/txd.md

```diff
@@ -0,0 +1,9 @@
+---
+description: TXD tools for Renderware Sonic games
+---
+# TXD Tools
+
+## TXD_Hack
+TXD texture archive editor writen with meticulous documentation.
```

Comment:
"meticulous documentation" doesn't sound too great. You can simplify this description by just typing this:

```
TXD texture archive editor.
```

Also, isn't there a better way to edit texture for these games nowadays? This seems really old.